### DISCUSSION TITLE
Guix Package for Lem

### DISCUSSION THREAD

**Discussion by jgarte** [lem-project/lem#1171] [Ideas]
Hi, 

This is a thread to open a discussion on the possibility of making a Guix package for lem. 

I'm starting to plan how best to approach this task.

## Questions

Does lem strictly depend on qlot currently to be built?

I know next to `nil` about qlot currently or how tightly coupled it is with the distribution of lem, itself.

Guix wraps `asdf` in a build-system that is implemented in GNU Guile:

https://github.com/guix-mirror/guix/blob/master/guix/build-system/asdf.scm

**Comment by spacelike** [comment]
It seems someone has already made an unofficial Guix package for it here: https://github.com/arthur-dog/lem-guix-packaging/blob/main/package.scm
I've tested it and it works very well.

**Reply by jgarte** [reply]
Cool, I know someone else that also recently made a Guix package for lem independently. I'll share to see if there is any work to consolidate.
